_includes/html/desktop-table.html

```diff
@@ -81,11 +81,19 @@
             <td class="icon-cell">
               <div class="row">
                 {% if tfa contains "custom-hardware" %}
-                  <i class="tfa-icon fas fa-check col"></i>
+                  <i class="tfa-icon fas fa-info col"
+                     title="{%- if website.custom-hardware -%}
+                     Required hardware token:&#013;{%- for item in website.custom-hardware -%} {{ item }}&#013; {%- endfor -%}
```

Comment:
This seems to be better, since it doesn't use the final line break:
```suggestion
                     Required hardware token:{%- for item in website.custom-hardware -%}&#013; {{ item }} {%- endfor -%}
```